src/core/output-layout.cpp

```diff
@@ -1744,6 +1754,11 @@ class output_layout_t::impl
 
         wlr_output_manager_v1_set_configuration(output_manager,
             wlr_configuration);
+
```

Comment:
This is the wrong place for this code, the function sends the wlr_output_manager configuration, not the wayfire signals.

In addition, wee need to send the configuration changed signal for each output (as you do it) but also the global output configuration changed signal for the whole output layout.

Reply:
I have moved it now, please check again.